File: 백준/Bronze/1978. 소수 찾기/소수 찾기.js
const fs = require('fs');
const io = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const [n, ...arr] = fs.readFileSync(io).toString().trim().split(/\s+/).map(Number);

function isPrime(value) {
  if (value === 1) return false;
  for (let i = 2; i <= Math.sqrt(value); i++) {
    if (value % i === 0) {
      return false;
    }
  }
  return true;
}

let cnt = 0;

arr.forEach((el) => {
  if (isPrime(el)) {
    cnt++;
  }
});

console.log(cnt);

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
import math

n = int(input())
arr= list(map(int, input().split()))
cnt = 0
for num in arr:
    if num == 1:
        continue
    for i in range(2, int(math.sqrt(num)) + 1):
        if num % i == 0:
            break
    else:
        cnt += 1

print(cnt)